skopt/optimizer/optimizer.py

```diff
@@ -172,7 +173,32 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
                 "Expected acq_optimizer to be 'lbfgs' or 'sampling', "
                 "got %s" % acq_optimizer)
 
-    def ask(self):
+    def ask(self, n_points=None, strategy="CL"):
+        """Query point or multiple points to evaluate.
+
+        * `n_jobs` [int or None]:
+            Number of function evaluations that can be done in parallel.
+            If the value is None, a single point to evaluate is returned.
+            Otherwise a list of points to evaluate is returned of size n_jobs.
+        * `strategy` [string]:
+            Strategy to use in order to query multiple points to evaluate.
+            Is ignored if n_jobs = None.
+        """
+        if n_points is None:
+            return self._ask()
+
+        opt = copy.deepcopy(self)
+        opt.rng = self.rng  # set random generator to not generate same points
+        x = []
+        for i in range(n_points):
+            x.append(opt.ask())
+            y_lie = np.min(opt.yi) if len(opt.yi) > 0 else 0.0  # CL-min lie
+            opt.tell(x[-1], y_lie)  # lie to the optimizer
+            self._n_random_starts = max(0, self._n_random_starts - 1)
```

Comment:
I call `ask` on copy of surrogate, not the surrogate itself. Thus I need to reduce this variable.